components/Common/ShellBox/__tests__/index.test.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { IntlProvider } from 'react-intl';
+import messages from '../../../../i18n/locales/en.json';
+import ShellBox from '../index';
+
+const mockWriteText = jest.fn();
+const originalNavigator = { ...window.navigator };
+
+describe('ShellBox', () => {
+  beforeEach(() => {
+    Object.defineProperty(window, 'navigator', {
+      value: {
+        clipboard: {
+          writeText: mockWriteText,
+        },
+      },
+    });
+  });
+
+  afterEach(() => {
+    Object.defineProperty(window, 'navigator', {
+      value: originalNavigator,
+    });
+  });
+
+  it('should render', () => {
+    const { container } = render(
+      <IntlProvider locale="en" messages={messages}>
```

Comment:
Remove `messages` from here (as per what we discussed) and pass `onError={() => {}}` to silence the errors from appearing in console.